packages/airnode-validator/src/cli/cli.ts

```diff
@@ -0,0 +1,63 @@
+import path from 'path';
+import { readFileSync } from 'fs';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { goSync } from '@api3/promise-utils';
+import ora from 'ora';
+import * as dotenv from 'dotenv';
+import { parseConfigWithSecrets } from '../api';
+
+export const succeed = (s: string) => ora(s).succeed();
+export const fail = (s: string) => ora(s).fail();
+
+const examples = [
+  '--config pathTo/config.json --secrets pathTo/secrets.env',
+  '-c pathTo/config.json -s pathTo/secrets.env',
+];
+
+export const validateConfiguration = (configPath: string, secretsPath: string) => {
+  const goRawConfig = goSync(() => readFileSync(path.resolve(configPath), 'utf-8'));
+  if (!goRawConfig.success) return fail(`Unable to read config file at "${configPath}". Reason: ${goRawConfig.error}`);
```

Comment:
None of these return statements will set an exit code and the CLI will finish with a 0 exit code.
![Screenshot_20220502_110631](https://user-images.githubusercontent.com/1655405/166211155-ba585b67-2336-4d90-811d-17573ab465e7.png)

Reply:
Done.